src/components/EditableField/EditableField.Input.tsx

```diff
@@ -326,7 +326,7 @@ export class EditableFieldInput extends React.Component<InputProps> {
             innerRef={this.setInputNode}
             name={name}
             placeholder={placeholder}
-            type={type === 'email' ? 'text' : type}
+            type="text"
```

Comment:
Would we ever use EditableField for something like a password field, where we would want the type to be `password` in order to obscure the text?

Reply:
Great point! I've made this change.